api/src/main/java/javax/jms/DupsOKAcknowledge.java

```diff
@@ -51,7 +51,7 @@
  * @see TopicListener
  * @see DupsOKAcknowledge
  *
- * @version JMS 2.1
+ * @version Jakarta Messaging 2.1
```

Comment:
There is no 2.1 version of this spec.  All the @version entries should be for 2.0.
In fact, this class should not even exist.  Ditto some of the others below.

Reply:
Good catch!  I was really thinking about that. the wrong code base must have been moved in by accident. Anyway, it should be fixed.